src/SFA.DAS.PAS.Account.Api.Types/ProviderAgreement.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.PAS.Account.Api.Types
+{
+    public class ProviderAgreement
+    {
+        public string Status { get; set; }
```

Comment:
We seem to have a number of approaches to how types are defined/exposed through clients and libraries.

In our legacy apps (such as this, although it looks as though this particular api only contains primitives), enums from the domain are reproduced/duplicated in the api type library and mapped across.

In our newer apps, we put our enums in a separate types library and reference it from the api types library to save the duplication.

Here we've gone another way - defining as a string. I understand you don't use the client for this api - do you take a reference to the types library? I guess not. If this implementation was change to match the legacy style - and a new enum was defined in the api types library - would this change anything for you? I believe it would still be rendered as a string in json.

Reply:
Cheers @chrisfoster76. I've now added a ProviderAgreementStatus enum to the API Types project.  I've also added a global Json formatter so enums are converted to string values in the API response. As this is the first enum that this service returns I think its safe enough to set globally.